jooby-netty/src/main/java/org/jooby/internal/netty/NettyResponse.java

```diff
@@ -184,11 +189,28 @@ public void send(final FileChannel channel, final long offset, final long count)
     rsp.headers().set(headers);
     ChannelHandlerContext ctx = this.ctx;
     ctx.channel().attr(NettyRequest.NEED_FLUSH).set(false);
+
+    // add chunker
+    ChannelPipeline pipeline = ctx.pipeline();
+    if (pipeline.get("chunker") == null) {
+      pipeline.addAfter("codec", "chunker", new ChunkedWriteHandler());
+    }
+
+    // Create the cunked input here already, to properly handle the IOException
+    final HttpChunkedInput chunkedInput = new HttpChunkedInput(new ChunkedNioFile(channel, offset, count, 8192));
```

Comment:
True, the main reason I put this outside the event loop was to be able to properly handle the IOException.